src/services/car/messageLead.ts

```diff
@@ -245,3 +246,75 @@ export const resendMessageLead = async ({
     result: {},
   }
 }
+
+export const fetchDealerWhatsappLeads = async ({
+  dealerId,
+  query,
+  options = {},
+}: {
+  dealerId: number
+  query: LeadQueryParams
+  options?: ApiCallOptions & { validateOnly?: boolean }
```

Comment:
```suggestion
  options?: ApiCallOptions
```

Why would you need `validateOnly` in the context of fetching a collection from the API?